InsightsFrontend/src/app/pages/release-graph/release-node.service.ts

```diff
@@ -234,13 +233,27 @@ export class ReleaseNodeService {
       );
       if (!synthetic) continue;
 
-      const baseX = synthetic.position.x + 75;
+      const baseX = synthetic.position.x + 35;
       const baseY = (index + 1) * ySpacing;
 
-      const positioned = subNodes.map((n, index_) => ({
-        ...n,
-        position: { x: baseX + index_ * xSpacing, y: baseY },
-      }));
+      const positioned: ReleaseNode[] = [];
```

Comment:
It might be nice for readability to fill this array with the result of another (new) method. Kind of to split the previous "checks" with the actual logic.